nominees/Women’s issues

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Women’s issues within political party platforms",
```

Comment:
@Rolikasi would like to propose an update of the project name to a shorter more catchy one which reflects the `GIthub repo` name, **Votes for women**. If you have any better reason for maintaining the current name, feel free to share. Thanks.

Reply:
Sounds great! I also renamed the file. Resolved.